validator/client/attest.go

```diff
@@ -79,6 +74,45 @@ func (v *validator) SubmitAttestation(ctx context.Context, slot uint64, pubKey [
 		}
 		return
 	}
+	indexedAtt.Signature = sig
```

Comment:
I think everything here can be abstracted into its own function to keep `SubmitAttestation` lean. It's basically check local then check remote, if error then return